geode-core/src/test/java/org/apache/geode/cache/client/internal/CacheServerSSLConnectionDUnitTest.java

```diff
@@ -178,11 +180,20 @@ public void setUpClientVM(String host, int port, boolean cacheServerSslenabled,
     gemFireProps.put(SERVER_SSL_CIPHERS, cacheServerSslciphers);
     gemFireProps.put(SERVER_SSL_REQUIRE_AUTHENTICATION, String.valueOf(cacheServerSslRequireAuth));
 
-    gemFireProps.put(SERVER_SSL_KEYSTORE_TYPE, "jks");
-    gemFireProps.put(SERVER_SSL_KEYSTORE, keyStorePath);
-    gemFireProps.put(SERVER_SSL_KEYSTORE_PASSWORD, "password");
-    gemFireProps.put(SERVER_SSL_TRUSTSTORE, trustStorePath);
-    gemFireProps.put(SERVER_SSL_TRUSTSTORE_PASSWORD, "password");
+
+    if (clientHasTrustedKeystore) {
+      gemFireProps.put(SERVER_SSL_KEYSTORE_TYPE, "jks");
```

Comment:
Do we have a corresponding test using the new SSL properties?

Reply:
good idea.